eslzArm/managementGroupTemplates/policyAssignments/DENY-PublicIpAddressPolicyAssignment.json

```diff
@@ -37,9 +30,17 @@
                 "description": "[variables('policyAssignmentNames').description]",
                 "displayName": "[variables('policyAssignmentNames').displayName]",
                 "policyDefinitionId": "[variables('policyDefinitions').denyPip]",
-                "enforcementMode": "[parameters('enforcementMode')]"
+                "enforcementMode": "[parameters('enforcementMode')]",
+                "parameters": {
+                    "listOfResourceTypesNotAllowed": {
+                        "value": [ "Microsoft.Network/publicIPAddresses" ]
```

Comment:
can we split this across multiple lines instead of a single line array (sorry picky I know but how we do it elsewhere)

Reply:
Suggestion?